_config.yml

```
@@ -19,14 +19,14 @@
 # in the templates via {{ site.myvariable }}.
 
 title: Bitcoin Design
-email: hi@bitcoindesign.com
+email: hi@bitcoindesign.org
```

We don't have an email server as far as I'm aware. If this property is needed though, perhaps best to suggest sender won't get a response.

```suggestion
email: no-reply@bitcoindesign.org
```